docker/bookkeeper/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update \
 &&  mkdir -p /opt \
 &&  cd /opt
 
-ENV BOOKKEEPER_VERSION=4.6.1 \
+ENV BOOKKEEPER_VERSION=4.6.2 \
```

Comment:
This seems wrong, why are we using `4.6+` here and `4.5.0` in  `gradle.properties`? The version should be consistent, no?

Reply:
The latest version is  `4.6.2`, but the integration tests are not yet ready to upgrade to it. They fail due to ` BK/ZK error`.
Previously, in the issue, when the bk version is upgraded to `4.6.1`, we agreed on having different versions for client `4.5.0`  and server `4.6.1`  in the case of deployment and system tests.  Same version of client and server `4.5.0` for integration tests.
ref :  https://github.com/pravega/pravega/pull/2348. 

The smoke test andrei mentioned in the above pr , passed with the latest version `4.6.2`.

Reply:
This is confusing. I don't want us to be running different versions like this, what needs to happen so that we make it uniform?

Reply:
#2480